packages/fx-core/src/plugins/resource/apim/constants.ts

```diff
@@ -399,3 +399,7 @@ export enum OpenApiSchemaVersion {
   V2 = "v2",
   V3 = "v3",
 }
+
+export enum UserTask {
```

Comment:
Do we really need an `enum` type here since we only have one element?

Reply:
Changed it to string.